cli/src/commands/mod.rs

```diff
@@ -275,6 +277,23 @@ fn make_branch_term(branch_names: &[impl fmt::Display]) -> String {
     }
 }
 
+/// Resolves revsets into revisions to rebase onto. These revisions don't have
+/// to be rewriteable.
+pub(crate) fn resolve_destination_revs(
```

Comment:
By the way (no action needed), this function seems reasonable to use from a custom binary, so we may want to make it public later.

Reply:
Happy to do that now, but it should also be renamed then, I think. `resolve_all_revs` perhaps?

Reply:
And it would probably be better somewhere further up the module hierarchy... Maybe just `cli_utils` again?

Reply:
SGTM. We can move it again if we find a better place later :)